composer.json

```diff
@@ -22,6 +22,9 @@
     "require": {
         "php": ">=5.4"
     },
+    "require-dev": {
+        "phpunit/phpunit": "~4.1.3"
```

Comment:
PHP 5.4 and 5.5 are not supported versions of PHP, so happy to go to minimum 5.6.